lib/dpl/provider/lambda.rb

```diff
@@ -147,6 +160,31 @@ def output_file_path
         @output_file_path ||= '/tmp/' + random_chars(8) + '-lambda.zip'
       end
 
+      def vpc_config
+        options[:subnet_ids] && options[:security_group_ids] ? { subnet_ids: options[:subnet_ids], security_group_ids: options[:security_group_ids] } : nil
+      end
+
+      def environment_variables
+        log "environment_variables: #{options[:environment_variables]}"
+        options[:environment_variables] ? { variables: options[:environment_variables] } : nil
+      end
+
+      def dead_letter_arn
+        options[:dead_letter_arn] ? {target_arn: options[:dead_letter_arn]} : nil
+      end
+
+      def tracing_mode
+        options[:tracing_mode] ? {mode: options[:tracing_mode]} : nil
+      end
+
+      def default_kms_key_arn
```

Comment:
It does not appear that this function and 2 below are invoked anywhere. Did I miss it?